22:55:48.349 [pool-62-thread-1] ERROR org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager - Error while checking the state of ledger re-replication
org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /ledgers/underreplication/disable
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:102) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:54) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.zookeeper.ZooKeeper.exists(ZooKeeper.java:2256) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.access$2301(ZooKeeperClient.java:70) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$13.call(ZooKeeperClient.java:833) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$13.call(ZooKeeperClient.java:827) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooWorker.syncCallWithRetries(ZooWorker.java:140) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.exists(ZooKeeperClient.java:827) ~[classes/:?]
	at org.apache.zookeeper.ZooKeeper.exists(ZooKeeper.java:2281) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.access$2401(ZooKeeperClient.java:70) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$14.call(ZooKeeperClient.java:854) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$14.call(ZooKeeperClient.java:848) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooWorker.syncCallWithRetries(ZooWorker.java:140) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.exists(ZooKeeperClient.java:848) ~[classes/:?]
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.isLedgerReplicationEnabled(ZkLedgerUnderreplicationManager.java:726) [classes/:?]
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.waitIfLedgerReplicationDisabled(ZkLedgerUnderreplicationManager.java:621) [classes/:?]
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.getLedgerToRereplicate(ZkLedgerUnderreplicationManager.java:602) [classes/:?]
	at org.apache.bookkeeper.replication.TestLedgerUnderreplicationManager$1.call(TestLedgerUnderreplicationManager.java:165) [test-classes/:?]
	at org.apache.bookkeeper.replication.TestLedgerUnderreplicationManager$1.call(TestLedgerUnderreplicationManager.java:162) [test-classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_292]
22:55:48.365 [pool-62-thread-1] ERROR org.apache.bookkeeper.replication.TestLedgerUnderreplicationManager - Error getting ledger id
org.apache.bookkeeper.replication.ReplicationException$UnavailableException: Error contacting zookeeper
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.isLedgerReplicationEnabled(ZkLedgerUnderreplicationManager.java:731) ~[classes/:?]
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.waitIfLedgerReplicationDisabled(ZkLedgerUnderreplicationManager.java:621) ~[classes/:?]
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.getLedgerToRereplicate(ZkLedgerUnderreplicationManager.java:602) ~[classes/:?]
	at org.apache.bookkeeper.replication.TestLedgerUnderreplicationManager$1.call(TestLedgerUnderreplicationManager.java:165) [test-classes/:?]
	at org.apache.bookkeeper.replication.TestLedgerUnderreplicationManager$1.call(TestLedgerUnderreplicationManager.java:162) [test-classes/:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_292]
Caused by: org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /ledgers/underreplication/disable
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:102) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:54) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.zookeeper.ZooKeeper.exists(ZooKeeper.java:2256) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.access$2301(ZooKeeperClient.java:70) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$13.call(ZooKeeperClient.java:833) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$13.call(ZooKeeperClient.java:827) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooWorker.syncCallWithRetries(ZooWorker.java:140) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.exists(ZooKeeperClient.java:827) ~[classes/:?]
	at org.apache.zookeeper.ZooKeeper.exists(ZooKeeper.java:2281) ~[zookeeper-3.6.2.jar:3.6.2]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.access$2401(ZooKeeperClient.java:70) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$14.call(ZooKeeperClient.java:854) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$14.call(ZooKeeperClient.java:848) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooWorker.syncCallWithRetries(ZooWorker.java:140) ~[classes/:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient.exists(ZooKeeperClient.java:848) ~[classes/:?]
	at org.apache.bookkeeper.meta.ZkLedgerUnderreplicationManager.isLedgerReplicationEnabled(ZkLedgerUnderreplicationManager.java:726) ~[classes/:?]
	... 8 more
